lib/project_types/node/commands/create.rb

```diff
@@ -113,9 +113,20 @@ def build(name)
         end
       end
 
+      def partners_url_for(organization_id, api_client_id)
+        if ShopifyCli::Shopifolk.acting_as_shopify_organization?
+          organization_id = 'internal'
+        end
+        "#{partners_endpoint}/#{organization_id}/apps/#{api_client_id}"
+      end
+
       def partners_endpoint
-        return 'https://partners.myshopify.io' if @ctx.getenv(ShopifyCli::PartnersAPI::LOCAL_DEBUG)
-        'https://partners.shopify.com'
+        domain = if @ctx.getenv(ShopifyCli::PartnersAPI::LOCAL_DEBUG)
+          'partners.myshopify.io'
+        else
+          'partners.shopify.com'
+        end
+        "https://#{domain}"
```

Comment:
Should these two methods actually be implemented in the PartnersAPI class?

Reply:
Yeah, maybe, I see we have other methods in there already which return URLs. I'll try it and see.

Reply:
f021ecc4